Tella/Data/Networking/APICall/URLRequest.swift

```diff
@@ -6,10 +6,13 @@
 import Foundation
 import Combine
 
+typealias APIResponse<Value> = AnyPublisher<(Value,[AnyHashable:Any]?), APIError>
```

Comment:
I think we could make the names more self explanatory to distinguish between `APIResponse` and `APIDataResponse`. Maybe using `APIValueResponse` could enhance the readability of the code.

Reply:
@ValbuenaG `Value` is a generic type, so we cannot rename the alias `APIValueResponse`